wp-sw-cache/lib/service-worker.js

```diff
@@ -1,61 +1,65 @@
-var CACHE_NAME = '$name';
+(function(self){
+  var CACHE_PREFIX = '__wp-sw-cache::';
 
-var CACHE_FILES = $files;
+  var CACHE_NAME = CACHE_PREFIX + $name;
```

Comment:
More, it is better to have `$name` instead of `'$name'` and having the string [JSON-encoded later](https://github.com/darkwing/wp-sw-cache/pull/15/files#diff-ec0a3d8373613972d86c8a140b9f7e61R32). This way mocking the SW is trivial as you can inject those variables as globals. :)